plextraktsync/media.py

```diff
@@ -43,10 +43,9 @@ def __init__(
 
     @property
     def title(self):
-        if self.plex:
-            return self.plex.title
-
-        return f"{self.trakt.title} ({self.trakt.year})"
+        if self.trakt:
+            return f"{self.trakt.title} ({self.trakt.year})"
+        return self.plex.title
```

Comment:
move this changeset to separate pr (with explanation etc)?

Reply:
This is the purpose of this PR.